datasets/il/mod_crypto/crawler.py

```diff
@@ -1,125 +1,135 @@
 import csv
 from typing import Dict
+from pathlib import Path
 
-from normality.cleaning import collapse_spaces
-
-from zavod import Context
-from zavod import helpers as h
+from zavod import Context, helpers as h
 from zavod.shed.zyte_api import fetch_html
 
+ID_FIELDS = [("id_no", "id_country"), ("residency_no", "residency_country")]
+LOCAL_PATH = Path(__file__).parent / "upstream_table.csv"
 
-def crawl_csv_row(context: Context, row: Dict[str, str]):
-    # Create a Person entity if present in the data
-    names = [
-        name
-        for name in [
-            collapse_spaces(row.get("English Name")),
-            collapse_spaces(row.get("Name Hebrew")),
-            collapse_spaces(row.get("Name Arabic")),
-        ]
-        if name != ""
-    ]
 
+def remove_zero_width_space(row):
+    return {
+        k: (v.replace("\u200b", "") if isinstance(v, str) else v)
+        for k, v in row.items()
+    }
+
+
+def check_csv_diff(context, container):
```

Comment:
(nit) Let's call this `write_csv_for_manual_diff` or something like this, doesn't really check anything.